lib/seed/runner.rb

```diff
@@ -69,6 +71,15 @@ def seed_patients(progress)
       results
     end
 
+    def seed_drug_stocks
+      Facility.all.each do |facility|
+        user = facility.users.first
+        facility.protocol.protocol_drugs.where(stock_tracked: true).each do |protocol_drug|
+          FactoryBot.create(:drug_stock, facility: facility, user: user, protocol_drug: protocol_drug)
```

Comment:
Sanity-check - let's ensure this factory call doesn't create extra tangential records like admins, organizations, etc.

Reply:
Looks okay to me. `facility`, `user`, and `protocol_drug` are the only associations on `DrugStock`.
Since we're providing all three here, there shouldn't be any unnecessary records created.